app/models/concerns/type_of_application.rb

```diff
@@ -13,10 +13,21 @@ def can_receive_pse?
   end
 
   def initial?
-    ApplicationType.new(application_type) == ApplicationType::INITIAL
+    application_type?(:initial)
   end
 
   def returned?
     returned_at.present?
   end
+
+  def post_submission_evidence?
+    application_type?(:post_submission_evidence)
+  end
+  alias pse? post_submission_evidence?
```

Comment:
nice, I was thinking about this with Hibos PR is "pse" as an acronymn a little too niche? I think because we all understand it then perhaps not but probs not the easiest thing for someone to grok who is new to the codebase. This like is a nice explainer in anycase 👍